crates/gitbutler-tauri/tauri.conf.json

```diff
@@ -1,7 +1,7 @@
 {
 	"build": {
-		"beforeDevCommand": "pnpm dev:internal-tauri",
-		"beforeBuildCommand": "[ $CI = true ] || pnpm build:desktop -- --mode development",
+		"beforeDevCommand": "cd ../ && pnpm dev:internal-tauri",
+		"beforeBuildCommand": "[ $CI = true ] || cd ../ && pnpm build:desktop -- --mode development",
```

Comment:
Why is this change required?

Reply:
Yeah good question, I don't remember exaclty but I assume with the additional `package.json` in the `crates/` dir, it ends up looking for the npm scripts there and fails to find them. Instead of looking one dir up as it normally would. Double checking now though :+1:

Reply:
Fair enough;

Would it make sense to use say `pushd ../ && pnpm <command>; popd`?

Reply:
Yeha looks like that's it :ok_hand: